app/controllers/application_controller/explorer.rb

```diff
@@ -221,7 +221,7 @@ def x_build_dynatree(options)
     # :full_ids               # stack parent id on top of each node id
 
     roots = x_get_tree_objects(options.merge(
-                                 :userid => session[:userid], # Userid for RBAC filtering
+                                 :user   => current_user,     # Userid for RBAC filtering
```

Comment:
thanks @h-kataria 
keep an eye open for `:userid => session[:userid]`. If possible, would like them to read `:user => current_user`